stan/math/prim/scal/fun/rising_factorial.hpp

```diff
@@ -51,11 +55,13 @@ namespace stan {
      */
     template<typename T1, typename T2>
     inline typename boost::math::tools::promote_args<T1, T2>::type
-    rising_factorial(const T1 x, const T2 n) {
-      using std::exp;
-      return exp(log_rising_factorial(x, n));
+    rising_factorial(const T1& x, const T2& n) {
```

Comment:
these should also get doxygen doc for arguments, return, exceptions, etc.  that goes for all the user-facing functions that get updated.  We're trying to clean them all up as we go.